packages/boilerplates/ScriptFailedError.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Dolittle. All rights reserved.
+*  Licensed under the MIT License. See LICENSE in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+export class ScriptFailedError extends Error {
```

Comment:
Postfix not needed. Read the exception guidance for C#. It is probably more specific than ScriptFailed.
Create specific exceptions - don't use generic exceptions. Also; we have a JavaScript fundamentals for exceptions: https://github.com/dolittle-fundamentals/JavaScript.Fundamentals/tree/master/Source/exceptions
Example of implementation: 
https://github.com/dolittle-fundamentals/JavaScript.Fundamentals/blob/master/Source/dependencyInversion/BindingTargetIsNotBasedOnFunction.js

When throwing: https://github.com/dolittle-fundamentals/JavaScript.Fundamentals/blob/master/Source/dependencyInversion/Container.js

Worth noting there is how we create a private method for actually doing the checking and throwing- making the code more readible. Look at the `throwIfBindingForServiceAlreadyExists` function at the top for instance. This can be done with private methods now in JavaScript and TypeScript.